test/integration/base.py

```diff
@@ -458,6 +485,43 @@ def assertTablesEqual(self, table_a, table_b,
             sql
         )
 
+    def assertManyTablesEqual(self, comparisons):
```

Comment:
have this take a list, and then recursively compare

```
def assertManyTablesEqual(self, tables, columns=None):
    schema = self.unique_schema()

    if columns is None:
        columns = self.get_table_columns_as_dict(tables, schema)

    first = tables[0]
    second = tables[1]
    rest = tables[1:]

    self.assertTablesEqual(first, second, schema, schema)
    self.assertManyTablesEqual(rest)
```

i think that accomplishes the same thing

Reply:
this is actually different - my original (badly written) version would take a dict of _multiple_ comparisons, eg:
```
assertManyTablesEqual({
  "seed": ["table_1", "table_2"],
  "incremental": ["table_3", "table_4"]
})
```

Here, a single query is used to fetch the columns for all 6 tables. The syntax you proposed would require two queries